app/src/test/java/edu/uc/brown5a7/BottomsUp/DrinkDTOMockkUnitTest.kt

```diff
@@ -28,34 +27,21 @@ class DrinkDTOMockkUnitTest {
 
     var drinkService = mockk<DrinkService>()
 
-    @Test
-    fun searchForCoke_returnsCoke(){
-        givenAFeedOfMockkedDrinkDataAreAvailable()
-        whenSearchForCoke()
-        thenResultContainsCoke()
-        thenVerifiedCokeFunctionsEnvoked()
-    }
-
-    private fun thenVerifiedCokeFunctionsEnvoked() {
-        verify { drinkService.fetchDrinks("coke") }
-        verify(exactly = 0) {drinkService.fetchDrinks("aadskfl") }
-        confirmVerified(drinkService)
-    }
-
-    private fun givenAFeedOfMockkedDrinkDataAreAvailable() {
+    @Before
+    fun setup()
+    {
         mvm = MainViewModel()
         createMockkData()
     }
 
     private fun createMockkData() {
-        var allDrinkLiveData = MutableLiveData<ArrayList<Drink>>()
-        var allDrinks = ArrayList<Drink>()
+        val allDrinkLiveData = MutableLiveData<ArrayList<Drink>>()
+        val allDrinks = ArrayList<Drink>()
 
-        //create mockk drink
-        var coke = Drink("26", "soda", "coca-cola", "coke", "Really good on a warm day!", "4")
+        val coke = Drink("26", "soda", "coca-cola", "coke", "Really good on a warm day!", "4")
         allDrinks.add(coke)
-        var MariageFreres = Drink("61", "Tea", "Marco Polo Blue", "Mariage Freres",  "My Favorite French Tea", "5")
-        allDrinks.add(MariageFreres)
+        val mariageFreres = Drink("61", "Tea", "Marco Polo Blue", "Mariage Freres",  "My Favorite French Tea", "5")
+        allDrinks.add(mariageFreres)
```

Comment:
As we never changes these values in the test, changed from the var type to the val type.

Reply:
Yes... this aligns well with Kotlin Coding Conventions.  https://kotlinlang.org/docs/reference/coding-conventions.htm